mscore/musescore.cpp

```diff
@@ -5665,6 +5665,7 @@ void MuseScore::endCmd()
             if (e && (cs->playNote() || cs->playChord())
                         && entryMethod != NoteEntryMethod::REALTIME_AUTO
                         && entryMethod != NoteEntryMethod::REALTIME_MANUAL) {
+                  cs->setPlayChord(true);
                   if (cs->playChord() && preferences.getBool(PREF_SCORE_CHORD_PLAYONADDNOTE) &&  e->type() == ElementType::NOTE)
```

Comment:
setting it to true and in the next line checking whether it is set to true seems... interesting?

Reply:
Well, my code explicitly specifies the desired behaviour and open ways for customisation (e.g. proposed by Marc).